workloads/kube-burner/workloads/large-networkpolicy-egress/case-networkpolicy-allowdns.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: case-amadeus-mgob-allow-dns
```

Comment:
@liqcui I searched and found 11 'amadeus', please remove or replace them.

Reply:
Fixed in new code, thanks!